src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf

```diff
@@ -7469,30 +7469,7 @@ Um die Warnung zu beheben, können Sie stattdessen /reference verwenden (Einbett
  /main:&lt;Typ&gt;                  Gibt den Typ an, der den Einstiegspunkt enthält
                                (alle anderen möglichen Einstiegspunkte werden ignoriert) (Kurz-
                                form: /m)
- /fullpaths                    Der Compiler generiert vollqualifizierte Pfade.
- /filealign:&lt;n&gt;                Gibt die Ausrichtung an, die für Ausgabedateiabschnitte
-                               verwendet werden soll.
- /pathmap:&lt;K1&gt;=&lt;V1&gt;,&lt;K2&gt;=&lt;V2&gt;,...
-                               Gibt eine Zuordnung für die Quellpfadnamen an, die vom 
-                               Compiler ausgegeben werden.
- /pdb:&lt;Datei&gt;                   Gibt den Namen der Debuginformationsdatei an (Standard:
-                               Ausgabedateiname mit der Erweiterung .pdb)
- /errorendlocation             Die Ausgabezeile und -spalte des Endpunkts
-                               jedes Fehlers 
- /preferreduilang              Gibt den bevorzugten Namen der Ausgabesprache an.
- /nostdlib[+|-]                Do not reference standard library (mscorlib.dll)
- /subsystemversion:&lt;Zeichenfolge&gt;    Verweist nicht auf die Standardbibliothek 
- /lib:&lt;Dateiliste&gt;              Gibt zusätzliche Verzeichnisse an, in denen nach Verweisen
-                               gesucht werden soll.
- /errorreport:&lt;Zeichenfolge&gt;         Gibt die Verarbeitung interner Compilerfehler an:
-                               "prompt", "send", "queue" oder "none". Der Standardwert ist
-                               "queue".
- /appconfig:&lt;Datei&gt;             Gibt eine Anwendungskonfigurationsdatei
-                               mit Assemblybindungseinstellungen an.
- /moduleassemblyname:&lt;Zeichenfolge&gt;  Der Name der Assembly, zu der dieses Modul
-                               gehören wird .
- /modulename:&lt;Zeichenfolge&gt;          Gibt den Quellmodulnamen an.
-</target>
+ /fullpaths                    Der Compiler generiert vollqualifizierte </target>
```

Comment:
@tmeschter It looks like the string got truncated on loc team's end somehow. Can you follow up on that?